telegram/client.py

```diff
@@ -382,6 +386,20 @@ def get_authorization_state(self) -> AsyncResult:
 
         return self._send_data(data, result_id='getAuthorizationState')
 
+    def _wait_authorization_result(self, result) -> str:
+        authorization_state = None
+        if result:
+            result.wait(raise_exc=True)
+
+            if result.update is None:
```

Comment:
`result.update` should always be presented, there is no need for this check

Reply:
See the previous comment and [python-telegram/client.py at line 421 on branch master](https://github.com/alexander-akhmetov/python-telegram/blob/master/telegram/client.py#L421)